packages/yourloops/pages/login/login.tsx

```diff
@@ -60,25 +59,31 @@ const loginStyle = makeStyles((theme: Theme) => {
     loginButton: {
       marginLeft: "auto !important",
     },
-    rightLink: {
-      padding: theme.spacing(0.5), // eslint-disable-line no-magic-numbers
-      textAlign: "start",
-      fontSize: "small",
-    },
-    centeredLink: {
-      padding: theme.spacing(0.5), // eslint-disable-line no-magic-numbers
+    Card: {
+      display: "flex",
+      flexDirection: "column",
+      justifyContent: "center",
       textAlign: "center",
-      color: "#109182",
+      // eslint-disable-next-line no-magic-numbers
+      padding: theme.spacing(4),
     },
-    leftLink: {
-      padding: theme.spacing(0.5), // eslint-disable-line no-magic-numbers
-      textAlign: "end",
-      fontSize: "small",
+    CardContent: {
+      textAlign: "start",
+      // eslint-disable-next-line no-magic-numbers
+      marginLeft: theme.spacing(4),
+      // eslint-disable-next-line no-magic-numbers
+      marginRight: theme.spacing(4),
     },
-    selection: {
+    CardActions: {
+      // eslint-disable-next-line no-magic-numbers
+      marginLeft: theme.spacing(4),
+      // eslint-disable-next-line no-magic-numbers
+      marginRight: theme.spacing(4),
```

Comment:
Maybe we can add 4 to the list of exception in .eslintrc ?

Or we can disable the this rule for the entire block